share/harvesters/io_osf.py

```diff
@@ -16,6 +16,10 @@ class OSFHarvester(BaseHarvester):
     VERSION = 1
 
     def build_url(self, start_date, end_date, path, query_params):
+        # so prod SHARE doesn't get throttled
+        if settings.OSF_BYPASS_THROTTLE_TOKEN:
+            self.requests.headers.update({'X-THROTTLE-TOKEN': settings.OSF_BYPASS_THROTTLE_TOKEN})
```

Comment:
Should probably use `self.session` -- `self.requests` is a rate-limited wrapper around `self.session`, so the `update()` will sort of have a throttle. Probably has no practical effect, but it will create a bunch of unnecessary wrappers as it is.